fixtures/malformedFile.JSON

```diff
@@ -0,0 +1,18 @@
+[
```

Comment:
This file is malformed because the object is expected to have a title and text key. The second has the  key "title" as "tile".